### ISSUE TITLE
Release worldwide, starting from ethnic minorities

### ISSUE THREAD

**Issue by kbobrowski** [corona-warn-app/cwa-wishlist#46]
I'm not sure if distribution of nationalities of CWA followers on GitHub correspond exactly to real distribution of minorities in Germany. Introduced voting system with likes to determine where app should be deployed is a nice gesture towards GitHub community, but perhaps more practical approach would be just to take a [official list of minorities](https://www.destatis.de/EN/Themes/Society-Environment/Population/Migration-Integration/Tables/foreigner-place-of-birth.html) and go through this list.

| Ethnic origin | Population | [Included in the list](https://github.com/corona-warn-app/cwa-backlog/issues/9#issuecomment-648740442) |
|------------------|----------------| -------------------------|
Turkey	|1,472,390 | :no_entry_sign: 
Poland	|862,535| :green_circle: 
Syria | 789,465 | :no_entry_sign: 
Romania|	748,225| :green_circle: 
Italy|	646,460 | :no_entry_sign: 
Croatia	|414,890 | :no_entry_sign: 
Greece	|363,650 | :no_entry_sign: 
Bulgaria	|360,170| :green_circle: 
Afghanistan |263,420 | :no_entry_sign: 
Russian Federation| 260,395 | :no_entry_sign: 
Iraq |255,050 | :no_entry_sign: 
Serbia (with/without Kosovo)	|237,755 | :no_entry_sign: 
Kosovo | 232,075 | :no_entry_sign: 
Hungary|	211,740 | :no_entry_sign: 
Bosnia and Herzegovina | 203,265 | :no_entry_sign: 
Austria	|186,725 | :green_circle: 
Spain	|177,755 | :no_entry_sign: 
Netherlands	|151,145| :green_circle: 
China |149,195 | :no_entry_sign: 
Ukraine |143,545 | :no_entry_sign: 
France	|140,290| :green_circle: 
Portugal	|138,410 | :no_entry_sign: 
United States |121,645 | :no_entry_sign: 
United Kingdom	|93,365 | :no_entry_sign:

**Comment by PivotNeilTarrant** [comment]
Whilst I appreciate the attempt to be representative beyond the GitHub community, I'm not sure that this list is much better.

The list here details foreigners across all time, so someone who moved here from Turkey in 1956 would be listed alongside someone who moved here six months ago. I would expect that people who have lived in Germany for many years - say from before the era of app stores - to only have accounts on the German store. 

Also do *all* countries have a native app store to target? Does Syria for instance? 

That said I'm not sure what would be a better source of data, perhaps to compare the above figures from 2015 (or 2010?) and see which countries have increased the most?

**Comment by fhajji** [comment]
Countries that don't have a specific Play Store usually default to the US Play Store. At least this one **really** deserves more love, if only for this reason alone.

**Comment by kbobrowski** [comment]
@PivotNeilTarrant agree that this neither is perfect representation of distribution of App Stores from different countries, wonder if there is official statistic about it.

As for native app stores to target - Google Play Store can target 149 counties and "rest of the world", the latter I guess includes Syria since it seems there is no native Play Store there.

**Comment by trevor87** [comment]
For those that want to install the android app in germany with a foreign account, here is a workaround:

https://github.com/corona-warn-app/cwa-app-android/issues/670

**Comment by NicolaiKilian** [comment]
It‘s not only ethnic minorities, it’s also e.g. Germans living in Germany but working in a neighboring country such as Switzerland, and are connected to the foreign App Store.

**Comment by kbobrowski** [comment]
@NicolaiKilian right, neighboring countries should also be high on priorities list

**Comment by phonique** [comment]
Here's a list of foreigners living in Germany. It's in German, the English version is paywalled. The list also includes a part of the population with migration background. If I understand correctly, these are the permanent residents (Aufenthaltstitel), 
https://de.statista.com/statistik/daten/studie/1221/umfrage/anzahl-der-auslaender-in-deutschland-nach-herkunftsland/

Maybe looking at incoming tourists should be another datapoint:
pp.18 et seq. contain the source markets of tourists in Germany for 2014:
https://www.germany.travel/media/pdf/dzt_marktforschung/DZT_Incoming-Tourismus-Deutschland-2015.pdf
Unfortunately, I was unable to find more up-to-date information disclosing an approximate distribution according to countries of origin.
